quickstarts/minikube.md

```diff
@@ -41,22 +76,32 @@ The above command might timeout if you're downloading images over a slow connect
 
 # Send and receive messages
 
-Once the cluster is running, you can run a simple producer to send messages to a Kafka topic (the topic will be automatically created):
+With the cluster running, run a simple producer to send messages to a Kafka topic (the topic is automatically created):
 
 ```shell
 kubectl -n kafka run kafka-producer -ti --image=quay.io/strimzi/kafka:{{site.data.releases.operator[0].version}}-kafka-{{site.data.releases.operator[0].defaultKafkaVersion}} --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server my-cluster-kafka-bootstrap:9092 --topic my-topic
 ```
 
-And to receive them in a different terminal you can run:
+And to receive them in a different terminal, run:
 
 ```shell
 kubectl -n kafka run kafka-consumer -ti --image=quay.io/strimzi/kafka:{{site.data.releases.operator[0].version}}-kafka-{{site.data.releases.operator[0].defaultKafkaVersion}} --rm=true --restart=Never -- bin/kafka-console-consumer.sh --bootstrap-server my-cluster-kafka-bootstrap:9092 --topic my-topic --from-beginning
 ```
 
 Enjoy your Apache Kafka cluster, running on Minikube!
 
+# Deleting your Apache Kafka cluster
+
+When you are finished with your Apache Kafka cluster, you can delete it by running:
+
+```shell
+kubectl -n kafka delete kafka my-cluster
+```
+
+This will remove the Apache Kafka cluster but leave the Strimzi cluster operator running so that it can respond to new Kafka custom resources.
+
```

Comment:
Should we have instructions for deleting the operator installation too, or deleting the minikube cluster?